modules/core/src/v2/coins/sol.ts

```diff
@@ -27,9 +29,9 @@ export interface TransactionFee {
 export type SolTransactionExplanation = TransactionExplanation;
 
 export interface ExplainTransactionOptions {
-  txHex?: string;
+  txBase64?: string;
   halfSigned?: {
-    txHex: string;
+    txBase64: string;
   };
```

Comment:
lets make `txBase64` a required string.
also, remove `halfSigned` for now. we'll add when needed